src/js/jsx/sections/libraries/LibraryBar.jsx

```diff
@@ -165,7 +170,22 @@ define(function (require, exports, module) {
 
         render: function () {
             var adobeStockURL = "https://stock.adobe.com";
-            
+
+            var addFillButton = this._createColorButton(function (layer) {
+                return layer.fills.isEmpty() ? null :
```

Comment:
It will be null here, same as standard PS. But we can definitely have a separate PR if we decide to show disabled button instead.